backend/chatbot.py

```diff
@@ -45,8 +45,8 @@ def checkAction(action):
     return action in ACTIONS
 
 def checkLocation(row,col):
-    return row > GRID_SIZE or col > GRID_SIZE \
-        or row < 0 or col < 0
+    return not (row > GRID_SIZE or col > GRID_SIZE \
```

Comment:
shouldn't it be GRID_SIZE - 1? if ur grid size is 3 ur highest index is 2.

Reply:
It should be